aws-redshift-cluster/src/test/java/software/amazon/redshift/cluster/CreateHandlerTest.java

```diff
@@ -287,4 +289,92 @@ public void testCreateClusterAndEnableLogging() {
                 .describeClusters(any(DescribeClustersRequest.class));
 
     }
+
+    @Test
+    public void testCreateCluster_MultiAZ() {
+        ResourceModel model = ResourceModel.builder()
+                .clusterIdentifier(CLUSTER_IDENTIFIER)
+                .masterUsername(MASTER_USERNAME)
+                .masterUserPassword(MASTER_USERPASSWORD)
+                .nodeType(NODETYPE)
+                .numberOfNodes(NUMBER_OF_NODES)
+                .clusterType("multi-node")
+                .allowVersionUpgrade(true)
+                .automatedSnapshotRetentionPeriod(0)
+                .encrypted(false)
+                .publiclyAccessible(false)
+                .clusterSecurityGroups(new LinkedList<String>())
+                .iamRoles(new LinkedList<String>())
+                .vpcSecurityGroupIds(new LinkedList<String>())
+                .enhancedVpcRouting(false)
+                .manualSnapshotRetentionPeriod(1)
+                .encrypted(true)
+                .multiAZ(true)
+                .build();
+
+        final ResourceHandlerRequest<ResourceModel> request = ResourceHandlerRequest.<ResourceModel>builder()
+                .desiredResourceState(model)
+                .region(AWS_REGION)
+                .logicalResourceIdentifier("logicalId")
+                .clientRequestToken("token")
+                .build();
+
+        when(proxyClient.client().createCluster(any(CreateClusterRequest.class)))
+                .thenReturn(CreateClusterResponse.builder()
+                        .cluster(MULTIAZ_CLUSTER)
+                        .build());
+
+        Cluster multiAZCluster = Cluster.builder()
+                .clusterIdentifier(CLUSTER_IDENTIFIER)
+                .masterUsername(MASTER_USERNAME)
+                .nodeType(NODETYPE)
+                .numberOfNodes(NUMBER_OF_NODES)
+                .clusterStatus("available")
+                .clusterAvailabilityStatus("Available")
+                .allowVersionUpgrade(true)
+                .automatedSnapshotRetentionPeriod(0)
+                .encrypted(true)
+                .multiAZ(MULTIAZ_ENABLED)
+                .enhancedVpcRouting(false)
+                .manualSnapshotRetentionPeriod(1)
+                .publiclyAccessible(false)
+                .clusterSecurityGroups(new LinkedList<ClusterSecurityGroupMembership>())
```

Comment:
can we make these as `Collections.emptyList()`

Reply:
Sure. I will send an updated PR using collections.emptyList().